### ISSUE TITLE
Make the `info` value in CentrifugePublicationEvent public

### ISSUE THREAD

**Issue by YoavGro** [centrifugal/centrifuge-swift#90]
Hi,

I'm using the swift SDK of Centerfuge and was wondering why the CentrifugePublicationEvent's info value is not public
Currently it looks like this

```
public struct CentrifugePublicationEvent {
    public var data: Data
    public var offset: UInt64
    public var tags: [String: String]
    var info: CentrifugeClientInfo?
}
```

Thanks

**Comment by FZambia** [comment]
Hello @YoavGro 

Not sure, seems like a miss - do you want to send PR? Also the same fix required for `CentrifugeServerPublicationEvent`

**Comment by YoavGro** [comment]
Sure, here is the [PR](https://github.com/centrifugal/centrifuge-swift/pull/91)

**Comment by FZambia** [comment]
@YoavGro many thanks! Released as part of [0.6.1](https://github.com/centrifugal/centrifuge-swift/releases/tag/0.6.1)

**Comment by YoavGro** [comment]
Amazing thanks!
